day-one-challenge/tyranny-of-the-rocket/getFuelRequirements.js

```diff
@@ -0,0 +1,20 @@
+const { sum, map } = require('ramda');
+const moduleMassArray = require('./fuelData');
+
+const calculateFuel = mass => {
+  return Math.floor(mass / 3) - 2;
+};
+
+const fuelOfCurrentModuleOfMass = (currentMass) => {
+  if (typeof (currentMass) !== 'number')
+    throw new TypeError("mass entered is not a number!");
```

Comment:
What happens when you put a _string_ into your data ("hello", for example). Does this guard catch it?

Reply:
When we add a string to the data and apply parseInt to the string, it will return NaN. I have included the logic to also catch NaN here: 99480b585e4036f563197c29bb96af1cadeebb1a